pkg/detectors/slackwebhook/slackwebhook.go

```diff
@@ -115,3 +116,14 @@ func (s Scanner) Type() detectorspb.DetectorType {
 func (s Scanner) Description() string {
 	return "Slack webhooks are used to send messages from external sources into Slack channels. If compromised, they can be used to send unauthorized messages."
 }
+
+func (s Scanner) IsFalsePositive(result detectors.Result) (bool, string) {
+	// ignore "https:" as a false positive for slack webhook detector
+	if strings.Contains(string(result.Raw), "https:") {
+		return false, ""
+	}
+
+	// back to the default false positive checks
+	return detectors.IsKnownFalsePositive(string(result.Raw), detectors.DefaultFalsePositives, true)
```

Comment:
When would this trigger? Won't the result always contain `https:`?

Personally, I think `https:` should be removed from the wordlist.

Reply:
Hmm, that can be done as well. For now it was for only one detector so I went with this approach.

Reply:
Sorry, the second sentence is a bit of a tangent.

I don't think the second part of the function will ever be reached.

Reply:
You’re absolutely right. I’ll simplify the function in next enhancement and remove the second part as it's unnecessary in this case. Appreciate the catch!